doc/api/events.markdown

```diff
@@ -4,8 +4,8 @@
 
 <!--type=module-->
 
-Many objects in Node.js emit events: a `net.Server` emits an event each time
-a peer connects to it, a `fs.readStream` emits an event when the file is
+Many objects in Node.js emit events: a [`net.Server`](net.html#net_class_net_server) emits an event each time
+a peer connects to it, a [`fs.readStream`](fs.html#fs_class_fs_readstream) emits an event when the file is
```

Comment:
while you are editing this line, could you please fix `fs.ReadStream` (capital R) ?